utils/build_grimoirelab

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA.
+#
+# Authors:
+#     Jesus M. Gonzalez-Barahona <jgb@bitergia.com>
+#
+
+import argparse
+import logging
+import os
+import os.path
+import subprocess
+import tempfile
+
+description = """Create packages for pypi.
+
+Example:
+    build_pypi.py
+
+"""
+
+build_dependencies = ['pip', 'setuptools', 'pypandoc', 'twine']
+install_dependencies = ['pip']
+
+#default_modules = ['grimoirelab-toolkit', 'perceval', 'perceval-opnfv',
+#        'perceval-mozilla', 'grimoireelk', 'sortinghat']
+default_commits = {
+        'grimoirelab-toolkit': 'HEAD',
+        'perceval': 'HEAD',
+        'perceval-opnfv': 'HEAD',
+        'perceval-mozilla': 'HEAD',
+        'grimoireelk': 'HEAD',
+        'sortinghat': 'HEAD'
+    }
+
+# Repos dictionary. For each git repository, if None, that
+# repo produces a single package, whose name is that of
+# the repo, and directory is the main repo directory.
+# If not None, list of package descriptions (package name and directory)
+repos = {
+    'grimoirelab-toolkit': None,
+    'perceval': None,
+    'perceval-opnfv': None,
+    'perceval-mozilla': None,
+    'grimoireelk': [{'name': 'grimoire-elk', 'dir': ''},
+                {'name': 'grimoire-kidash', 'dir': 'kidash'}],
+    'sortinghat': None
+}
+
```

Comment:
Maybe default_commits and repos could me merged to avoid the same entries (git repositories) duplicated in two places? Trying to follow the DRY pattern (don't repeat yoursefl).

Reply:
Thanks for the comment, @acs. This is a good idea. I split the information in two dictionaries because of their different origins. In one case (`default_commits`) it was just a fallback for the case when the user does not define a coordinated release file. In the other (`repos`) it was "structural" information about repos and packages.

Both could be joined together in a single dictionary, but I would like to explore another option. That would be using the format in `default_commits` for the coordinated release. In any case, both dictionaries could be merged in future versions of this script.

This said, if you prefer, I can merge dictionaries in this pull request, that shouldn't be that difficult.

Reply:
Just go ahead if you are going to change it. The repositories are not going to change frequently and both dictionaries are pretty close in the code, to maintain them. And this is a script, so it won't grow too much.